tests/wpt/metadata/css/css-transforms/animation/perspective-interpolation.html.ini

```diff
@@ -0,0 +1,2 @@
+[perspective-interpolation.html]
+  expected: CRASH
```

Comment:
Hmm, it shows up again 🤔

Reply:
Yeah, I forgot to mark #24239 with the right label.